data/json/items/ammo_types.json

```diff
@@ -281,6 +281,18 @@
     "name": "gasoline",
     "default": "gasoline"
   },
+  {
+    "type": "ammunition_type",
+    "id": "jp8",
+    "name": "jp8",
+    "default": "jp8"
+  },
+  {
+    "type": "ammunition_type",
+    "id": "avgas",
+    "name": "avgas",
+    "default": "avgas"
+  },
```

Comment:
I'm passingly familiar with aviation (dad was a hobby pilot). Neat to see aviation fuel types making it into the game! A few questions, mostly around game design:

1. Do we want to allow players to use gas of the incorrect quality for some application? I can see somebody using kerosene or car gas for a hot air balloon heater, but doing so for an aviation engine should cause some sort of problems. If we do enable this, we may want to think about some sort of engine-fouling mechanic. This could also make things like makeshift drive-belts more relevant.

2. The wikipedia article on [Jet Fuel](https://en.wikipedia.org/wiki/Jet_fuel) has all sorts of fun info on various JP-X blends that were made from a variety of other mixtures, some of which could be relevant to CDDA:
    * JP-1 was more or less purified kerosene with a high flash point. This could be a nice path towards makeshift jetfuel
    * JP-2 was JP-1, but with a higher freezing point. Do we have engine freezing mechanics? This could be relevant if we ever do.
    * JP-3 is basically JP-2 with a _very_ permissive distillation process. Wikipedia has a great quote on it: \
      > In his book Ignition! An Informal History of Liquid Rocket Propellants, John D. Clark described the specification as, "remarkably liberal, with a wide cut (range of distillation temperatures) and with such permissive limits on olefins and aromatics that any refinery above the level of Kentucky moonshiner's pot still could convert at least half of any crude to jet fuel".
    
      After reading that I think I'll buy _Ignition_ and see if for no other reason than some fun flavor test too contribute. :joy:

    * JP-4 is a 50-50 kerosene-gasoline blend. These are both fuels that are easy to find and make.

There's a lot of other good stuff in the article, but I think the above are the only relevant fuels for the purposes of powering aircraft engines/turbines.

Reply:
Sure, engine fouling eventually maybe, turbines in-game can currently run on a number of fuels, some would be sub-optimal irl, out of scope of this PR though.